examples/exp_configs/templates/sumo/test.net.xml

```diff
@@ -0,0 +1,5727 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
should be deleted. also two above xml files should be deleted. @AboudyKreidieh ?